en/animation/joint-texture-layout.md

```diff
@@ -0,0 +1,88 @@
+
+The following figure shows a sample scenario with multiple instances from the same model, playing completely different animations at the same time. As you can see, the current scene, plus the UI, has a total Drawcall of 60 and an instance count of 0. This state will be used as the basis for later changes to compare.
+
+![Baseline](./joint-texture-layout/instancing_baseline.gif)
+
+To create a model with instancing version turned on, the following steps are required:
+
+1. In the **Inspector** panel, check the `UseBakedAnimation` property in the SkeletalAnimation component (using pre-baked animation mode).
+
+    ![use baked animation](./joint-texture-layout/use-baked-animation.png)
+
+2. check `USE INSTANCING` for all materials used by SkinningModel.
+
+    ![Enabling](./joint-texture-layout/enabling_instancing.png)
+
+In our example scene we actually made two sets of Prefab and set the material diffuse color to blue in the instancing version in order to see and distinguish the performance of both systems. You can see that the effect has been exactly correct, and that only 5 Drawcalls (each model is divided into 5 parts) were used, with 45 instances.
+
+![Normal](./joint-texture-layout/instancing_normal.gif)
+
+> **Note**: the reason for rendering all models correctly here is that the amount of animation data is still relatively small, and the generic joint texture global reuse logic already writes all animation data to the same texture, so the effect is correct. However, if the size of the default joint texture (360 * 360) is exceeded by new animations that may be added at any time, the animation will definitely go wrong, which is why the joint texture layout panel must exist.
+
+## Joint Texture Layout Effects
+
+For demonstration purposes, we can intentionally put each animation on a separate texture in the **Joint Texture Layout Settings** panel to see the final rendering effect.
+
+First, open the **Joint Texture Layout Settings** panel:
+
+![Panel](./joint-texture-layout/joint_texture_layout_new.png)
+
+The three **+** in the panel are used as follows:
+
+- ① - Used to add Texture units, a Texture unit consists of multiple Skeleton units.
+- ② - Used to add Skeleton units, a Skeleton unit consists of a Skeleton asset and one to more AnimationClip assets.
+- ③ - Used to add AnimationClip asset slots.
+
+Here we have 9 different animations separated in 9 Texture cells.
+
+![WrongLayout](./joint-texture-layout/joint_texture_layout_wrong.png)
+
+Rerun the scene and the effect becomes:
+
+![Wrong](./joint-texture-layout/instancing_wrong.gif)
+
+You can see that there is a problem with the animation, all the animations become attack actions, and the model disappears from time to time. The reason behind this can be precisely analyzed.
```

Comment:
```suggestion
Notice there is a problem with the animation, all the animations become attack actions, and the model disappears from time to time. The reason behind this can be precisely analyzed.
```